app/src/interfaces/presentation-text-block/index.ts

```diff
@@ -0,0 +1,31 @@
+import { defineInterface } from '@directus/shared/utils';
+import InterfacePresentationTextBlock from './presentation-text-block.vue';
+import PreviewSVG from './preview.svg?raw';
+
+export default defineInterface({
+	id: 'presentation-text-block',
+	name: '$t:interfaces.presentation-text-block.text-block',
```

Comment:
In the process of working on the theming interface I had the need for a simple text block interface (This is how I add the inline section descriptions). This is the only interface I've added that isn't scoped to the `_system/system-theming` folder. It's a very simple interface, just add some text and it displays on the page. 

With that said, while I think the interface could be useful to have on the user-facing side, I'm not sure if it's suitable in it's current state. So I wanted to pass off the question and see if you think I should leave this interface as it is, or move it into the `_system/system-theming` folder?